src/index.ts

```diff
@@ -84,10 +85,17 @@ function createExtensions(document, result, requestID, userAgent) {
     ? fetchLoggerRequestDone(requestID, userAgent)
     : {}
 
+  // Should this introspect xapp token/roles for `email_provider` instead?
```

Comment:
I wondered this same thing. It would be more reliable, but more complex code-wise. Overall I think it's OK to use the user-agent in this way because Braze is a well-behaved integration. The worst case is that a caller spoofing their user-agent gets this same, conservative treatment.

Reply:
Exactly, worst-case winds up being this is applied more often/overzealously. I think it's fine that we try to call it 'email provider' in our code, but somewhere in the stack (ideally maybe kept to one or minimal spots), there is a check that winds up having to mention 'braze'.

Reply:
You may not know and this is a bit of a off topic question but I've wondered for a while about the ID thats part of Braze user-agent:
```
Braze Sender 954a8af2d130500d09e736a5e8ce1db59da5c25f; Metaphysics
```
What is the purpose of that ID? Is that something that is unique for the organization that Braze has a relationship with or perhaps its masking information to make it harder to spoof the user-agent? Does the ID stay constant or can it change at any point? 🤔